prog-o-meter.py

```diff
@@ -318,8 +375,10 @@ def main():
     name_screen = UsernameGUI(user_state)
     username = name_screen.get_name()
     filename = "".join((username.lower(), ".txt"))
+    logname = "".join((username.lower(), "_log.txt"))
     if user_state == 2:        #Make a new file for a new user, and set their current progress to 0 days
         update_days_file(filename, "0")
+        open(logname, 'w').close()        #Make a new log file for a new user
     days = read_days_file(filename)
     days = int(days)
     ProgressGUI(days, filename, username)

```

Comment:
program can't run, because we are now supposed to pass 4 args to ProgressGUI, but we only pass 3.

Reply:
Added the fourth argument, "logname" to the ProgressGUI call here.